core/types/istanbul_test.go

```diff
@@ -63,12 +64,9 @@ func TestExtractToIstanbul(t *testing.T) {
 				},
 				AddedValidatorsPublicKeys: [][]byte{},
 				RemovedValidators:         big.NewInt(12), //1100
-				Bitmap:                    big.NewInt(0),
-				Seal:                      []byte{},
```

Comment:
nit:  Was removal of the `Seal` field unintentional?

Reply:
Nope!